### PR TITLE
Filter projects to exclude non-Jira ones

### PR DESCRIPTION
**Pull request by sebbASF**

### CODE REVIEW

**Comment by Humbedooh** [comment]
I am somewhat wary about this, as it will increase the level of spam. Accumulo received a bunch of spam requests primarily because they are listed first alphabetically. If we remove them from the dropdown (as opposed to displaying an error when you submit, as happens now), we are just moving the problem to the next project on the list.

File: server/app/endpoints/public.py [hunk 1]
```diff
@@ -22,8 +22,13 @@
 
 
 async def process(form_data):
+    print(form_data)
```
Comment (martin-g) [inline server/app/endpoints/public.py:25]: debug leftover ?!

File: server/app/lib/config.py [hunk 2]
```diff
@@ -42,6 +42,24 @@
...
+  "cordova",
+  "couchdb",
+  "fluo",
+  "kibble",
+  "linkis",
+  "lucene",
+  "netbeans",
+  "skywalking",
+  "trafficcontrol",
+  "trafficserver",
```
Comment (martin-g) [inline server/app/lib/config.py:60]: What about the Bugzilla ones ?
Tomcat and HTTPD also need to reject requests often.

**Comment by Humbedooh** [comment]
For projects that do not use Jira, we have an internal list. Account requests submitted against those will immediately inform the requester that this project does not use Jira.  Genuine applicants will read this message and adjust their request if it is in earnest.

Leaving out projects from the drop-down would, in my opinion, cause confusion for folks that then can't find the project they are looking for. 

We can look at alternate or better ways of informing the user that the project does not use Jira, but we should not filter them out altogether.

**Pull request closed without merging**